<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>popup_window</class>
 <widget class="QDialog" name="popup_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>244</width>
    <height>122</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>244</width>
    <height>122</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>244</width>
    <height>122</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Scanning...</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
